src/__tests__/components/ArticleCard.test.js

```diff
@@ -1,13 +1,30 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
+import renderer from 'react-test-renderer';
 import ArticleCard from '../../components/Article/ArticleCard';
 import { articleData } from '../../__mocks__/dummyData';
 
-const props = { article: articleData, classes: '' };
+let wrapper;
+const props = {
+  loading: true,
+  article: articleData,
+  url: '/articles/article-slug',
+  history: { push: jest.fn() },
+};
 
 describe('<ArticleCard />', () => {
+  beforeEach(() => {
+    wrapper = mount(<ArticleCard {...props} />);
+  });
```

Comment:
@NkFab Can you add an empty line after this line for better readability?

Reply:
Done